taskcluster/scripts/bitrise-schedule.py

```diff
@@ -136,25 +158,33 @@ async def schedule_build(client, branch, commit, workflow, locales=None, derived
     return response["build_slug"]
 
 
-async def wait_for_job_to_finish(client, build_slug):
-    suffix = f"builds/{build_slug}"
+async def wait_for_job_to_finish(client, build_slug, pipeline=False):
+    if pipeline:
+        suffix=f"pipelines/{build_slug}"
+    else:
+        suffix = f"builds/{build_slug}"
+
     url = BITRISE_URL_TEMPLATE.format(suffix=suffix)
 
     while True:
         response = await do_http_request_json(client, url)
-        if response["data"]["finished_at"]:
-            log.info(f"Job {build_slug} is now finished, checking result...")
+        if pipeline:
+            log.info(f"Job {build_slug} is a pipeline TBD what to do next")
```

Comment:
We don't have an end point in Bitrise to retrieve the log file or the artifacts directly for pipelines. Once we decide how to manage the artifacts generated and what to store, we will implement that part accordingly